scripts/garage.coffee

```diff
@@ -41,17 +49,31 @@ module.exports = (robot) ->
 	getGarages = (callback) ->
 		r = request 'https://secure.parking.ucf.edu/GarageCount/iframe.aspx/', (error, response, body) ->	
 			garages = []
-			$ = cheerio.load(body);
+			pubsub = false;
+			$ = cheerio.load(body)
 			$('.dxgvDataRow_DevEx').each (i, obj) ->
-				thisGarage = {};
+				thisGarage = {}
 				html = $(obj).html().replace(RegExp(' ', 'g'), '').split '\n'
 				for line in html
 					if line.startsWith("percent:")
 						percent = parseInt(line.replace("percent:", ''))
 						thisGarage.perc = percent
 				thisGarage.garage = ($(obj).find('.dxgv').html()).replace("Garage ", '')
 				garages[i] = thisGarage
-			callback garages
-		return null
-
-
+			
+			if robot.brain.get('pubsubCheckDate') isnt (new Date()).toDateString()
+				robot.brain.set('pubsubCheckDate', (new Date()).toDateString())
```

Comment:
This is more a matter of opinion, but wouldn't it be better to assign today's date to a variable instead of calling `(new Date()).toDateString()` twice? It's not a performance thing tho, more a readability thing.